assessor/lib/testcore.rb

```diff
@@ -0,0 +1,26 @@
+
+def test_clear(engine)
+  set errexit: false do
+    run "cf marketplace"
+    stdout, _, _ = capture "cf service-brokers"
+    matches = stdout.match(/(minibroker-\S*)/)
+    if matches
```

Comment:
```suggestion
    if status.success? && matches
```
(Where we hold on to the exit status from `capture`)

Reply:
Fixed with commit dc68b80856848585810d27f16be4ba547b0cd313.